src/test/kotlin/racingcar/MovableTest.kt

```diff
@@ -1,32 +1,23 @@
 package racingcar
 
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
 
 internal class MovableTest {
 
-    @Test
-    fun `4 미만의 값(0)은 전진할 수 없다`() {
-        assertThat(Movable().canGo(0)).isEqualTo(false)
+    @DisplayName("0 ~ 3 범위의 수 테스트")
+    @ParameterizedTest(name = "4미만의 값({0})는 전진할 수 없다")
```

Comment:
```suggestion
    @ParameterizedTest(name = "4미만의 값({0})는 멈춘다")
```
"전진할 수 없다." 와 "전진할 수 있다."는 자세히 봐야 있다 없다의 차이를 눈으로 확인할 수 있어요.
이런 경우에는 단어를 다르게 작성하면, 더 눈에 잘 들어오게 돼요. 😄